### ISSUE TITLE
Pokemon Ultra Sun — Low FPS issues

### ISSUE THREAD

**Issue by brlamio** [citra-emu/citra#4615]
**Issue**
I am currently running the latest version of Citra Canary on macOS Mojave Version 10.14.2. Having dealt with some significant FPS drop running Pokemon Ultra Sun. Title screen tends to run at maximum performance between (50-55fps). Lags and stutters starts when I am navigating around the environments in-game and in certain cut scene transition between player and NPC. From then moving forward in game, it drops to a steady (1-5fps) becoming unplayable at that point.

**System Specs**
Processor: 2.8 GHz Intel Core i7
Memory: 16GB 1600 MHz DDR3
Graphics: AMD Radeon R9 M370X 2048 MB / Intel Iris Pro 1536 MB
Citra Version: Citra Canary 1183
Game: Pokemon Ultra Sun

**Diagnostic Log**
[citra_log.txt](https://github.com/citra-emu/citra/files/2818966/citra_log.txt)

**Screenshots**
<img width="512" alt="screen shot 2019-01-31 at 12 36 03 pm" src="https://user-images.githubusercontent.com/13665690/52084202-808f8e00-256f-11e9-85d6-17e3c6016277.png">

<img width="512" alt="screen shot 2019-01-31 at 12 38 43 pm" src="https://user-images.githubusercontent.com/13665690/52084222-8be2b980-256f-11e9-9fac-6036aae337b2.png">

**Comment by SleepingSnakezzz** [comment]
Citra macOS OpenGL builds are no longer supported due to Citra now requiring a higher OpenGL version. Which Apple does not support. Once Vulkan is merged, and this issue still persists, please open up a new issue. Closing.
